test/choosetests.jl

```diff
@@ -15,7 +15,7 @@ Upon return, `tests` is a vector of fully-expanded test names, and
 """ ->
 function choosetests(choices = [])
     testnames = [
-        "linalg", "core", "keywordargs", "numbers", "printf",
+        "interactiveutil", "linalg", "core", "keywordargs", "numbers", "printf",
```

Comment:
This shouldn't be first thing, sorry - linalg needs to stay first, and it's better for other time-consuming tests to be towards the start wherever possible. You can run just these tests by themselves via `make test-interactiveutil`